tests/sync/courseInstancesSync.js

```diff
@@ -9,10 +10,18 @@ const helperDb = require('../helperDb');
 const { assert } = chai;
 
 describe('Course instance syncing', () => {
-  before('remove the template database', helperDb.dropTemplate);
+  // before('remove the template database', helperDb.dropTemplate);
```

Comment:
leftover commented-out code (and similarly elsewhere)

Reply:
These are intentionally left in here, these need to be run every time the DB changes when running individual tests with `npx mocha path/to/file.js`. It was easiest to leave them there and uncomment when needed.

Reply:
sounds good! maybe a comment to that effect, if this is going to be a long-term thing?